templates/funcs_st2chatops.jinja

```diff
@@ -0,0 +1,62 @@
+###############[ ST2CHATOPS ]###############
+{% if id in ["rhel", "rocky", "centos", "opensuse-leap"] -%}
+nodejs_configure_repository()
+{
+    curl -fsSL https://rpm.nodesource.com/setup_20.x | sudo -E bash -
```

Comment:
This is fun. If this script is supposed to be used for multiple st2 versions then we would need to install different nodejs versions based on the st2 version. 😅

It is sad that we're downloading and running a script that might not have been reviewed by the user, but they can skip installing st2chatops, so I guess that's ok. Or at least, it doesn't change the status quo.